recipes/hankel/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "hankel" %}
+{% set version = "0.3.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6c2c0ba5143efd6556e657349ca7927e0db3821a23e3458a3748f71c792de2a5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - scipy >=0.12.0
+    - mpmath >=0.18
+    - numpy >=1.6.1
+    - future
+
+test:
+  imports:
+    - hankel
+
+about:
+  home: "https://github.com/steven-murray/hankel"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.rst
+  summary: "Hankel Transformations using method of Ogata 2005"
+  description: |
+    Hankel transforms and integrals are commonplace in any area in which Fourier Transforms are required over fields that are radially symmetric.
+    They involve integrating an arbitrary function multiplied by a Bessel function of arbitrary order (of the first kind).
+    Typical integration schemes often fail because of the highly oscillatory nature of the transform.
+    Ogata's quadrature method used in this package provides a fast and accurate way of performing the integration based on locating the zeros of the Bessel function.
+  doc_url: https://hankel.readthedocs.io/
+  dev_url: https://github.com/steven-murray/hankel
+
+extra:
+  recipe-maintainers:
+    - MuellerSeb
+    - steven-murray
```

Comment:
@steven-murray are you happy to be listed as a maintainer if this?

Reply:
@chrisburr : since there is no reply, I have shortened the maintainerlist for now.

Reply:
If it's not too late, yes I'm happy :-) Sorry for the late reply!

Reply:
@MuellerSeb, you can add @steven-murray in a PR to the feedstock.